src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -39,6 +39,8 @@ public static class ScriptConstants
         public const string LogPropertyProcessIdKey = "ProcessId";
         public const string LogPropertyActivityIdKey = "MS_ActivityId";
 
+        public const string IgnoreApplicationInsightsKey = "MS_IgnoreApplicationInisghts";
```

Comment:
Is this used anywhere? I don't see a reference.

Reply:
Not any more; i've removed it in my next commit  -- also spelled wrong!